other/n-Queen.py

```diff
@@ -0,0 +1,90 @@
+# Python3 program to solve N Queen  
+# Problem using backtracking 
+global N 
+N = 4
+  
+def printSolution(board): 
+    for i in range(N): 
+        for j in range(N): 
+            print (board[i][j], end = " ") 
```

Comment:
Please remove the space between __print__ and __(__.  It is a function just like __range()__.